### PR TITLE
fix cors issue

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request fixes the CORS issues.

### Approvals

- [x] Dev

### CODE REVIEW

File: onboarding-api/src/index.ts [hunk 1]
```diff
@@ -2,5 +2,23 @@ import * as functions from 'firebase-functions'
 import { businessVerificationController } from './controllers/businessVerification'
 import { businessVerificationConfirmController } from './controllers/businessVerificationConfirm'
 
-exports.businessVerification = functions.https.onRequest(businessVerificationController)
-exports.businessVerificationConfirm = functions.https.onRequest(businessVerificationConfirmController)
+const allowedDomains = ['http://localhost:3000']
```
Comment (sophialittlejohn) [inline onboarding-api/src/index.ts:5]: could you add port 8888 here? IDK why yours runs on port 3000 🤯

**Review by sophialittlejohn — APPROVED** [review]
Nice job!!!

File: onboarding-api/.eslintrc.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  ignorePatterns: ['lib/**'],
+  ignorePatterns: ['lib/**', 'dist/**'],
```
[the remark this answers is not in the document]
Comment (jpangelle) [reply onboarding-api/.eslintrc.js]: removed

File: onboarding-api/src/index.ts (hunk 1 above)
Comment (jpangelle) [reply onboarding-api/src/index.ts]: made generic for any localhost port

**Pull request merged**